README.md

```diff
@@ -250,6 +250,16 @@ the [Lombok Gradle Plugin](https://plugins.gradle.org/plugin/io.freefair.lombok)
 to delombok your source code before it is passed to the Checker Framework
 for typechecking. This plugin does not support any other use of Lombok.
 
+## Compatibility with other annotation processors
+
+The plugin runs checkers by creating a custom manifest file containing the 
```

Comment:
This first sentence is true, but is not what the section is about, and therefore is a bad way to start it.  Begin with what the reader cares about, which is the second sentence.  Delete the first sentence, which is more appropriate for developer documentation than the user manual.